docs/authentication.md

```diff
@@ -18,16 +18,31 @@ This doc will cover how to set this up!
 
 The Chains controller will use the same service account your Task runs under as credentials for pushing signatures to an OCI registry. This section will cover how to set up a service account that has the necessary credentials.
 
-First, you will need access to credentials for your registry (they are in a file called `credentials.json` in this example).
-Next, create a [Docker config type Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/#docker-config-secrets), which will contain the credentials required to push signatures:
-
 Set the namespace and name of the Kubernetes service account:
 
 ```shell
 export NAMESPACE=<your namespace>
 export SERVICE_ACCOUNT_NAME=<service account name>
 ```
 
+### Create a Secret based on existing credentials
+
+If you already ran `docker login`, you can copy the credentials into Kubernetes:
```

Comment:
Can we add a step here where users confirm that the ~/.docker/config.json file is in this format?

```
{
  "auths": {
    "https://index.docker.io/v1/": {
      "auth": "redacted"
    }
}
```

on mac the dockerconfig uses keychain so it usually isn't in this format

Reply:
@priyawadhwa makes sense! Update, PTAL 🙃